test/fixer/sqs_client_test.rb

```diff
@@ -0,0 +1,62 @@
+# encoding: utf-8
+
+require 'minitest_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.